### PR TITLE
Update github-merge-jenkins to start logging errors to Rollbar

### PR DESCRIPTION
**Pull request by nicoecheza**
We want to start logging erros to Rollbar in this Rollbar's [project](https://rollbar.com/tactivos/github-merge-jenkins).

**NOTE**: The secret for `ROLLBAR_ACCESS_TOKEN` has already been added.

### CODE REVIEW

**Comment by carmon** [comment]
LGTM

File: github-merge-jenkins.js [hunk 1]
```diff
@@ -31,16 +35,21 @@ module.exports = function githubMergeJenkins(context, callback) {
...
+      request({ 
+        url, 
+        method: 'POST', 
+        form: { action, branch } 
+      }, (err, res, body) => callback(err, body));
+
     } else {
-      console.log(`Ignoring hook because branch is '${branch}' and action is ${action}`);
+      const msg = `Ignoring hook because branch is '${branch}' and action is ${action}`;
+      rollbar.reportMessageWithPayloadData(msg, { body: context.body, params });
```
Comment (pzavolinsky) [inline github-merge-jenkins.js]: I suggest keeping the `console.log` and the `rollbar` notifier.

File: github-merge-jenkins.js [hunk 2]
```diff
@@ -31,16 +35,21 @@ module.exports = function githubMergeJenkins(context, callback) {
...
       return callback();
     }
   }
   else {
-    console.log(`Ignoring hook for repo ${context.body.repository.full_name}; no action or branch name specified`);
+    const msg = `Ignoring hook for repo ${context.body.repository.full_name}; no action or branch name specified`;
+    rollbar.reportMessageWithPayloadData(msg, { body: context.body, params });
```
Comment (pzavolinsky) [inline github-merge-jenkins.js]: Same as above.

File: github-merge-jenkins.js (hunk 1 above)
Comment (nicoecheza) [inline github-merge-jenkins.js]: Discussed it with patoz and concluded `console.log` is unnecessary.

Comment (pzavolinsky) [inline github-merge-jenkins.js]: Nevermind both comments

File: github-merge-jenkins.js (hunk 2 above)
Comment (nicoecheza) [inline github-merge-jenkins.js]: Same as above.

**Pull request merged**